mklove/modules/configure.base

```diff
@@ -342,13 +641,27 @@ $(mkl_var_get MKL_FAIL__${n}__reason)
 "
         # Dig up some metadata to assist the user
         case $MKL_DISTRO in
-            Debian|Ubuntu|*)
+            debian|ubuntu)
                 local debs=$(mkl_meta_get $conf "deb")
-                pkg_cmd="sudo apt-get install"
+                pkg_cmd="sudo apt install -y"
                 if [[ ${#debs} > 0 ]]; then
                     install_pkgs="$install_pkgs $debs"
                 fi
                 ;;
+            centos|rhel|redhat|fedora)
+                local rpms=$(mkl_meta_get $conf "rpm")
+                pkg_cmd="sudo yum install -y"
+                if [[ ${#rpms} > 0 ]]; then
+                    install_pkgs="$install_pkgs $rpms"
+                fi
+                ;;
+            alpine)
+                local apks=$(mkl_meta_get $conf "apk")
+                pkg_cmd="apk install "
```

Comment:
Is there a `-y` equivalent for apk ?

Reply:
No, it is non-interactive it seems.
More importantly, there is no "install", it is called "add"